lib/user_app_jwt/Kconfig

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig USER_APP_JWT
+	bool "User Application JWT Library"
+	select BASE64
+	select CJSON_LIB
+	# needed to print integer values in json
+	select CBPRINTF_FP_SUPPORT
+
+if USER_APP_JWT
+
+config JWT_VERIFY_SIGNATURE
+	bool "Verify signature after signing"
+	default y
+
+config PRINT_EXPORTED_PUBKEY_DER
+	bool "Print to terminal the DER formatted public key"
+	default y
+
+config USER_APP_JWT_MAX_LEN
```

Comment:
This is used only in the sample application, this does not belong in the library configuration.